CodeEdit/NavigatorSidebar/SourceControlNavigator/Views/Changes/ChangesView.swift

```diff
@@ -0,0 +1,47 @@
+//
+//  ChangesView.swift
+//  CodeEdit
+//
+//  Created by Nanashi Li on 2022/05/05.
+//
+
+import SwiftUI
+import CodeEditUI
+import Git
+
+struct ChangesView: View {
+
+    @ObservedObject
+    var model: SourceControlModel
```

Comment:
nit: can this be changed to a constant?

Reply:
Property wrapper can only be applied to a 'var'